src/cdk/explorer/nodes/getCfnDefinition.ts

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import { getLogger, Logger } from '../../../shared/logger'
+
+/**
+ * @param {string} uniqueIdentifier - unique identifier of state machine
+ * @param {string} templatePath - path for the template.json file
+ * 
+ * @returns the escaped ASL Json definition string of the state machine construct
+ */
+export function getStateMachineDefinitionFromCfnTemplate(uniqueIdentifier: string, templatePath: string) {
+    const logger: Logger = getLogger()
+    try {
+        let data = fs.readFileSync(templatePath, 'utf8')
+        const jsonObj = JSON.parse(data)
+        const resources = jsonObj.Resources
+
+        for (const key of Object.keys(resources)) {
+            if (key === 'CDKMetadata') continue
+
+            const slicedKey = key.slice(0, -8)
```

Comment:
I don't think it's safe for us to assume the suffix is a fixed length. We may need to write some logic to find a list of keys that are prefixed with our identifier, and then have some sort of tie-breaking logic after that.